include/wx/dc.h

```diff
@@ -1405,6 +1405,78 @@ class WXDLLIMPEXP_CORE wxDCTextColourChanger
     wxDECLARE_NO_COPY_CLASS(wxDCTextColourChanger);
 };
 
+// ----------------------------------------------------------------------------
+// helper class: you can use it to temporarily change the DC text background colour and
+// restore it automatically when the object goes out of scope
+// ----------------------------------------------------------------------------
+
+class WXDLLIMPEXP_CORE wxDCTextBgColourChanger
+{
+public:
+    wxDCTextBgColourChanger(wxDC& dc) : m_dc(dc), m_colBgOld() { }
+
```

Comment:
Calling `m_colBgOld()` makes no difference and does no have to be here: the default ctor would be called automatically anyway.